backend/models/viewpoint.js

```diff
@@ -52,8 +52,6 @@ const clean = function(routePrefix, viewpointToClean, serialise = true) {
 
 		if (serialise) {
 			setViewpointScreenshotURL(routePrefix, viewpointToClean);
-			viewpointToClean.screenshot_ref = undefined;
```

Comment:
@Charence Can You take a look at this branch? Removing this line, I've fixed the issue with updating the viewpoint without passing a new screenshot. However, I broke the backend tests. Probably this line was here for a reason, but in my cases, its absence helps. When is this entry needed?

Reply:
This is the internal reference on where the screenshot is stored, it shouldn't ever be exposed to the public domain so removing this should be fine. The backend probably have golden data that contains it and thus erroring out. I'll have a look. Thanks!